index.bs

```diff
@@ -3165,40 +3190,28 @@ the same procedure as other [=assertion signatures=] generated by the [=authenti
         </thead>
         <tbody>
             <tr>
-                <th> Second-factor platform authenticator </th>
+                <th> <dfn>Second-factor platform authenticator</dfn> </th>
```

Comment:
There's the U2F implementation Intel Online Connect... I don't know whether that supports UV or not, but I'd guess probably not since it's based on U2F.